### PR TITLE
Rename /lookup endpoint to /reputation

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: packages/pangea-sdk/pangea/services/intel.py [hunk 1]
```diff
@@ -46,7 +47,7 @@ class FileLookupResult(PangeaResponseResult):
     raw_data: Optional[Dict] = None
 
 
-class IPLookupRequest(APIRequestModel):
```
Comment (govind-salinas-pangea) [inline packages/pangea-sdk/pangea/services/intel.py:49]: in other instances you are keeping the old data types around, why not do that here?

Comment (pangea-andrest) [reply packages/pangea-sdk/pangea/services/intel.py:49]: Python wouldn't notice it. Other instances are compiled and if we change return type would crash.
Also, in Python, this class is used just internally. In other instances it's exposed to the user

**Review by govind-salinas-pangea — APPROVED** [review]

**Comment by r3dcrosse** [comment]
Should we rename all the lookup methods for the python sdk?

I currently know about:
```
file_intel.lookup
file_intel.lookupFilepath

domain_intel.lookup

ip_intel.lookup

url_intel.lookup
```

but should we be doing:
```
file_intel.lookup -> file_intel.reputation
file_intel.lookupFilepath -> file_intel.filepathReputation

domain_intel.lookup -> domain_intel.reputation

ip_intel.lookup -> ip_intel.reputation

url_intel.lookup -> url_intel.reputation
```
?

**Comment by r3dcrosse** [comment]
The `generate_docs` gitlab job is failing with the error:
```
$ poetry run python parse_module.py > python_sdk.json
Traceback (most recent call last):
  File "parse_module.py", line 227, in <module>
    docs = parse_pangea()
  File "parse_module.py", line 218, in parse_pangea
    import pangea.services
  File "/builds/pangeacyber/pangea-python/packages/pangea-sdk/pangea/services/__init__.py", line 3, in <module>
    from .intel import DomainIntel, FileIntel, IpIntel, UrlIntel
  File "/builds/pangeacyber/pangea-python/packages/pangea-sdk/pangea/services/intel.py", line 6, in <module>
    from deprecated import deprecated
ModuleNotFoundError: No module named 'deprecated'
```

I think to fix, we need to add deprecated to the `pyproject.toml`? Can make it go away by doing:
```
poetry add Deprecated
```
from within `pangea-python/packages/pangea-sdk`

**Pull request merged**